.gitignore

```diff
@@ -21,6 +21,10 @@ _testmain.go
 
 *.exe
 
+# Backup files
+*~
+
```

Comment:
I'm aware of that. The problem is though, that you can't rely on everybody else's setup to be correct - hence why I personally tend to keep backup and swap files in a repo's .gitignore. Just my 2 cents, will remove it for now.